torch/_torch_docs.py

```diff
@@ -5719,14 +5719,25 @@
 
 add_docstr(torch.zeros,
            r"""
-zeros(*sizes, out=None) -> Tensor
+zeros(size, out=None, dtype=None, layout=torch.strided, device=None, requires_grad=False) -> Tensor
```

Comment:
Is the tuple a requirement (I.e., a breaking change), or can we still pass variable number of ints? I'm not mentioning we should change the doc, but it's for my understanding actually :-)